### ISSUE TITLE
api.MouseWheelEvent - Not supported in Chrome/Safari

### ISSUE THREAD

**Issue by saschanaz** [mdn/browser-compat-data#8226]
#### What information was incorrect, unhelpful, or incomplete?

BCD says it's supported.

#### What did you expect to see?

Proper `removed_version` values.

#### Did you test this? If so, how?

A simple existence check.

#### Can you link to any release notes, bugs, pull requests, or MDN pages related to this?

https://developer.mozilla.org/en-US/docs/Web/API/MouseWheelEvent

**Comment by ddbeck** [comment]
This seems right, but with a caveat.

I couldn't find any evidence that the `MouseWheelEvent` name was ever implemented in WebKit. However, I did find that the `wheelDelta*` properties from `MouseWheelEvent` were implemented as [properties on the `WheelEvent` interface](https://github.com/WebKit/WebKit/blob/88278b55563e5ccdc0b3419c6c391c3becc19e40/Source/WebCore/dom/WheelEvent.idl#L37-L40), which is a bit odd (Chrome also supports these legacy properties).

I think the right fix here might be to mark `MouseWheelEvent` as unsupported in Safari, but add the non-standard features to `WheelEvent` at the same time.
